src/core/client/ui/components/ValidationMessage/ValidationMessage.css

```diff
@@ -0,0 +1,35 @@
+.root {
+  composes: validationMessage from "talk-ui/shared/typography.css";
+  position: relative;
+  display: inline-flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 7px 10px;
+  box-sizing: border-box;
+}
+
+.colorRegular {
+  background-color: transparent;
+  border: none;
+  color: var(--palette-common-black);
+  padding-left: 40px;
+}
+
+.colorError {
+  background-color: var(--palette-error-light);
+  border-radius: 1px;
+  border-color: var(--palette-error-darkest);
+  border-left-width: 8px;
+  border-left-style: solid;
+  color: var(--palette-common-white);
+}
+
+.fullWidth {
+  display: block;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.icon {
+  margin-right: 8px;
```

Comment:
Hm weird spacing here, will need to ask @IAmSamHankins if this shouldn't be 1 * spacing-unit